client/src/components/Jobs/Jobs.fixture.js

```diff
@@ -5,10 +5,10 @@ const jobsList = [
     owner: 'username',
     systemId: 'test.exec.system',
     appId: 'test-app',
-    created: '2020-05-01T14:44:46.000-05:00',
```

Comment:
can you add a test like:
`expect(getByText("05/01/2020 09:44")).toBeDefined();`
to `client/src/components/Jobs/Jobs.test.js`

Reply:
Note:  this was tested so can just be added.